app/views/dashboard/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class='container'>
+  <div class='container'>
+    <h1>Favorite 3 Companies:</h1><br>
+  </div>
+  <div class='col-md-6 well'>
+  <% @company_interests.each do |interest, company| %>
+    <h4>
+      <p> <%= link_to company, company_path(Company.find_by(name: company)) %> has an average level of interest of:<%= interest %></p>
```

Comment:
we want to avoid querying our database in our views. where else could we do `Company.find_by(name: company)`?